### PR TITLE
Preserve Music JSON Data

### PR DESCRIPTION
**Pull request by AsimKhan2019**
Completed the operations of adding the music data and the genre data against the music. Still working on the JSON data.

### CODE REVIEW

**Comment by AsimKhan2019** [comment]
@Bria222 There seems to be an issue with the JSON file creation, it does not create a JSON file and add it via the preserver module. Can you suggest what could be the problem?

**Comment by AsimKhan2019** [comment]
@Bria222 I added the JSON files, now they are showing data.

**Comment by AsimKhan2019** [comment]
@Bria222 The work is complete, please review and let me know

File: classes/genre.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    @id = Random.rand(1...700)
+    @name = name
+    @items = []
+  end
+
+  def add_items(item)
+    @items << item
+    item.genre = self
+  end
+end
```
Comment (Bria222) [inline classes/genre.rb:15]: nice class implimentation

File: json/genres.json [hunk 2]
```diff
@@ -0,0 +1 @@
+[{"genre_name":"hiphop"}]
```
Comment (Bria222) [inline json/genres.json:1]: the logic works fine

**Review by Bria222 — APPROVED** [review]
Hi @AsimKhan2019 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

STATUS: APPROVED 🟢
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Bria222  in your question so I can receive the notification.

**Pull request merged**